tools/osbuilder/rootfs-builder/ubuntu/config.sh

```diff
@@ -32,4 +32,4 @@ INIT_PROCESS=systemd
 ARCH_EXCLUDE_LIST=()
 
 [ "$SECCOMP" = "yes" ] && PACKAGES+=" libseccomp2" || true
-[ -n "$SKOPEO_UMOCI" ] && PACKAGES+=" ca-certificates libgpgme11" || true
+[ -n "$SKOPEO" ] && PACKAGES+=" ca-certificates libgpgme11" || true
```

Comment:
Again, shouldn't this be testing for `"yes"` explicitly?

Reply:
Updated. Thank you!